CONTRIBUTING.md

```diff
@@ -39,44 +39,24 @@ Hard tabs are required rather than spaces.
 ## Releasing a new OCM API Model version
```

Comment:
The section title is talking about "release" but the description underneath( the last part) is talking more about updating/bumping the model in the SDK.

I wonder how applicable is this section to the ocm-sdk-go repo. I find that it has its place in the ocm-api-model.